cmake/SSGCommon.cmake

```diff
@@ -641,6 +641,16 @@ macro(ssg_make_html_stats_for_product PRODUCT)
     )
 endmacro()
 
+macro(ssg_render_policies_for_product PRODUCT CONTROL_FILES)
+        foreach(CONTROL_FILE IN LISTS CONTROL_FILES)
+        add_custom_target(${PRODUCT}-render-policy-${CONTROL_FILE}
```

Comment:
I think this should use `add_custom_command` instead, as there is an output file:
https://cmake.org/cmake/help/latest/command/add_custom_target.html

I can run `ninja render-policies` indefinitely and regenerate the artifacts, although there was no change to the DS.

Reply:
I've switched to `add_custom_command` instead, but it builds nevertheless if it's requested or not. Does it make sense to build this type of content every time we build the project?

Reply:
removing the ALL from the custom target did the trick.